rdkit/Chem/Fraggle/UnitTestFraggle.py

```diff
@@ -6,24 +6,28 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
-from rdkit import RDConfig
-import unittest, os
-from rdkit.six.moves import cPickle
+import doctest
+import unittest
+
 from rdkit import Chem
 from rdkit.Chem.Fraggle import FraggleSim
+from rdkit.Chem.Fraggle.FraggleSim import select_fragments
 
 
-class TestCase(unittest.TestCase):
+def load_tests(loader, tests, ignore):  # pylint: disable=unused-argument
```

Comment:
If I understand this correctly it is loading the doctests from FraggleSim.py and they will be executed. If that's the case, then FraggleSim.py should be removed from test_list.py